TST: Cosmology broke CI

Can't tell from the run history if it is from #12740 or #12741.

Example logs:

* https://github.com/astropy/astropy/actions/runs/1709793538
* https://github.com/astropy/astropy/runs/4848331765?check_suite_focus=true

Example error:

```
______________ Testw0wzCDM.test_is_equivalent_to_row[astropy.row] ______________

self = <astropy_tests.cosmology.tests.test_flrw.Testw0wzCDM object at 0x7f99ec7d87c0>
cosmo = w0wzCDM(name="ABCMeta", H0=70.0 km / (Mpc s), Om0=0.27, Ode0=0.73, w0=-1.0, wz=0.5, Tcmb0=3.0 K, Neff=3.04, m_nu=[0. 0. 0.] eV, Ob0=0.03)
to_format = <astropy.cosmology.connect.CosmologyToFormat object at 0x7f99ec7320a0>
format = 'astropy.row'

>   ???
E   NameError: name 'Cosmology' is not defined

astropy/cosmology/io/tests/test_row.py:113: NameError
```

@nstarman , any ideas?

--- a/astropy/cosmology/io/tests/test_model.py
+++ b/astropy/cosmology/io/tests/test_model.py
@@ -10,7 +10,7 @@ import pytest
 import numpy as np
 
 # LOCAL
-from astropy.cosmology.core import _COSMOLOGY_CLASSES
+from astropy.cosmology.core import _COSMOLOGY_CLASSES, Cosmology
 from astropy.cosmology.io.model import _CosmologyModel, from_model, to_model
 from astropy.cosmology.tests.conftest import get_redshift_methods
 from astropy.modeling import FittableModel

--- a/astropy/cosmology/io/tests/test_row.py
+++ b/astropy/cosmology/io/tests/test_row.py
@@ -4,7 +4,7 @@
 import pytest
 
 # LOCAL
-from astropy.cosmology.core import _COSMOLOGY_CLASSES
+from astropy.cosmology.core import _COSMOLOGY_CLASSES, Cosmology
 from astropy.cosmology.io.row import from_row, to_row
 from astropy.table import Row
 

